utils.py

```diff
@@ -39,7 +39,8 @@ def parse_beatmap_file(content: str) -> Dict[str, dict]:
     current_section = None
 
     for line in content.splitlines():
-        line = line.strip()
+        # this line breaks storyboards
```

Comment:
You don't need to comment this one, just remove it ;)

Reply:
ok